### PR TITLE
fix(Pixie): Update docs to highlight new Pixie entities

### PR DESCRIPTION
**Pull request by htroisi**
Pixie Entities were recently added to the New Relic UI. This PR updates the docs to highlight these new entities.

### CODE REVIEW

File: src/content/docs/kubernetes-pixie/auto-telemetry-pixie/advanced-configuration/manage-pixie-data.mdx [hunk 1]
```diff
@@ -8,13 +8,17 @@ tags:
...
-metaDescription: 
+metaDescription:
 redirects:
   - /docs/auto-telemetry-pixie/manage-pixie-data
 ---
 
 You have options for the type and the amount of data you ingest when you install Auto-telemetry with Pixie. During install, use Helm to reduce, restrict, or exclude data, by excluding specific namespaces or pods, by collecting data for only the nodes you want, or by redacting columns that contain sensitive information.
 
+<Callout variant="important">
+The following sections show how to configure reduced ingest during installation of Pixie. You can also reduce ingest after installation by [disabling select Pixie export scripts](/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/overview/#pixie-configure-storage).
```
Comment (nserrino) [inline src/content/docs/kubernetes-pixie/auto-telemetry-pixie/advanced-configuration/manage-pixie-data.mdx:19]: i would emphasize configure over reduce here. people might want to add new export scripts

File: src/content/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/explore-pixie-data.mdx [hunk 2]
```diff
@@ -220,3 +225,36 @@ The **See TCP retransmissions** and **See dropped packets** links will show grap
 </figcaption>
 
 Note that the above **Kubernetes network monitoring with Pixie** views can also be seen in the [Live debugging with Pixie](/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/live-debugging-with-pixie) tab using the `px/dns_flow_graph`, `px/tcp_drops` and `px/tcp_retransmits` scripts. These scripts have optional arguments that allow you to filter the output to select entities.
+
+## Inspect full-body requests [#inspect-requests]
+
+You can use Pixie to inspect the full-body requests and responses made by your applications for [select protocols](/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/overview/#pixie-long-term-storage).
+
+Prototocls traced by Pixie show up under the **Other entities** section of the **All entities** page:
```
Comment (nserrino) [inline src/content/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/explore-pixie-data.mdx:233]: typo: protocols

**Review by nserrino — DISMISSED** [review]

File: src/content/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/explore-pixie-data.mdx [hunk 3]
```diff
@@ -46,7 +46,7 @@ Auto-telemetry collected by Pixie can be found throughout New Relic's UI. Use Pi
 * [Monitor networking issues](#network-health)
 * [Inspect full-body requests](#inspect-requests)
 
-New Relic’s UI pulls Pixie telemetry data from both short-term and long-term storage. Find out more about the hybrid storage model for the New Relic Pixie integration [here](/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/overview).
+The UI of New Relic pulls Pixie telemetry data from both short-term and long-term storage. Find out more about the hybrid storage model for the New Relic Pixie integration [here](/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/overview).
```
Comment (htroisi) [inline src/content/docs/kubernetes-pixie/auto-telemetry-pixie/understand-use-data/explore-pixie-data.mdx:49]: @akristen I prefer the previous wording. Could we revert the changes here?

**Comment by htroisi** [comment]
@akristen would it be possible to keep the original `pixie_screenshot-full_dns_entity.png` image? I want to show the graphs as well as the tables.

**Comment by nbaenam** [comment]
> 

@htroisi, I've added a new image, please let me know if this one it's OK. The original `pixie_screenshot-full_dns_entity.png` image is not OK according to our guidelines. Thanks!
